options.c

```diff
@@ -75,6 +75,8 @@ ExpandOptions (void)
 {
     ExpandString (o.exe_path, _countof(o.exe_path));
     ExpandString (o.config_dir, _countof(o.config_dir));
+    ExpandString (o.global_config_dir, _countof(o.global_config_dir));
+    ExpandString (o.config_auto_dir, _countof(o.config_auto_dir));
```

Comment:
Probably its wrong to expand these strings as these are restricted locations and a limited user should not be allowed to override them using env vars. Same with install_path below (not touched by this patch).. Right?